pipelines/push-to-external-registry/push-to-external-registry.yaml

```diff
@@ -4,7 +4,7 @@ kind: Pipeline
 metadata:
   name: push-to-external-registry
   labels:
-    app.kubernetes.io/version: "0.21.0"
+    app.kubernetes.io/version: "1.21.0"
```

Comment:
This is not really a breaking change, right? Similar to Johnny's https://github.com/redhat-appstudio/release-service-catalog/pull/225 where he also doesn't increase the major version. Because it's ok to provide extra parameters to a pipeline - they just won't be used.

Reply:
Changed all versions.